hack/test-end-to-end.sh

```diff
@@ -178,7 +178,7 @@ sudo env "PATH=${PATH}" OPENSHIFT_ON_PANIC=crash openshift start \
 OS_PID=$!
 
 if [[ "${API_SCHEME}" == "https" ]]; then
-	export CURL_CA_BUNDLE="${CERT_DIR}/master/root.crt"
```

Comment:
I know this is end-to-end, and we're making assumptions, but technically, the trusted root bundle for the master is what we really want... there's no guarantee the CA signing cert (if we have one) was used to generate the server cert.